### PR TITLE
New ISSUE_TEMPLATE.md file #1637

### PR DESCRIPTION
**Pull request by DevinWalker**
## PR Overview

Added an `ISSUE_TEMPLATE.md` file as a baseline. This is the one we use in our projects and it has been working pretty nicely for us. 

If you have any suggested edits, just let me know! 

Ref: #1637

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Issue Overview
```
Comment (ntwb) [inline .github/ISSUE_TEMPLATE.md]: Probably above this should be a link [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Current Behavior
+<!-- If describing a bug, tell us what happens instead of the expected behavior -->
+<!-- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!-- Not obligatory, but suggest a fix/reason for the bug, -->
+<!-- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
```
Comment (ntwb) [inline .github/ISSUE_TEMPLATE.md]: I'd reorder this, the sections should be in the order of:

`## Steps to Reproduce (for bugs)`
`## Expected Behavior`
`## Current Behavior`

It allows the issue reader to gain an overview of the events that led up to the "current behaviour" they see

**Review by ntwb — CHANGES_REQUESTED** [review]
Also see #1637

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (DevinWalker) [inline .github/ISSUE_TEMPLATE.md]: That makes sense to me!

**Comment by youknowriad** [comment]
Should we add a comment asking to look for duplicates before opening an issue?

**Comment by DevinWalker** [comment]
@youknowriad yes, that's definitely a good idea. I'll add that text.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+<!--
+BEFORE POSTING YOUR ISSUE:
+- These comments won't show up when you submit the issue.
+- Try to add as much detail as possible. Be specific!
+- If you're requesting a new feature, explain why you'd like it to be added.
+- Search this repository for the issue and whether it has been fixed or reported already.
+- Ensure you are using the latest code before logging bugs.
+- Disabled all plugins to ensure it's not a plugin conflict issue.
```
Comment (swissspidy) [inline .github/ISSUE_TEMPLATE.md]: s/Disabled/Disable

**Pull request merged**